leonardo-example.conf

```diff
@@ -0,0 +1,119 @@
+# Example config file for leonardo.
+# Anything commented out here will have have defaults specified or is not necessary to run leo.
+# All values can be modified for your specific google environment, database, and security certificates
+
+
+# Google Cloud dataproc configuration
+dataproc {
+  #applicationName = "leonardo"
+  dataprocDefaultRegion = "YOUR_REGION"  # the google region for your dataproc
+  leoGoogleProject = "GOOGLE_PROJECT"    # the name of the google project to use during cluster startup
+                                         # *this is not the project the cluster will be created in
+  clusterUrlBase = "https://YOUR_DOMAIN/notebooks" # the base url to access your cluster
+  #jupyterServerName = "jupyter-server"
```

Comment:
It isn't clear what this config option does, even after reading the code I'm hazy on it's role as a config option.

Reply:
This is just the name of the docker container that runs Jupyter on the dataproc master node. It's probably not something that users would care to configure; maybe it should be moved to `reference.conf`.